tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -6205,6 +6285,30 @@ Result D3D12Device::createQueryPool(const IQueryPool::Desc& desc, IQueryPool** o
     }   
 }
 
+Result D3D12Device::createFence(const IFence::Desc& desc, IFence** outFence)
+{
+    RefPtr<FenceImpl> fence = new FenceImpl();
+    SLANG_RETURN_ON_FAIL(fence->init(this, desc));
+    returnComPtr(outFence, fence);
+    return SLANG_OK;
+}
+
+Result D3D12Device::waitForFences(
```

Comment:
I'm assuming the timeout is in milliseconds - because thats what WaitForMultipleObjects takes, but it doesn't seem to be documented as a comment in the interface, or as part of the parameter name.

Reply:
Yeah, I need to make sure the timeOut paameter means the same thing across d3d and vulkan here.